scripts/tests/localsetup_tests.py

```diff
@@ -1214,6 +1185,216 @@ def test_start_unsupported_version(self, _ignore_load_images):
+            setup = LocalSetup(argv=self.common_setup_args + ["master", "--all", "--dyno"])
+            setup.set_docker_compose_path(docker_compose_yml)
+            setup()
+        docker_compose_yml.seek(0)
+        got = yaml.safe_load(docker_compose_yml)
+        self.assertIn('dyno', got['services'])
+
+    @mock.patch(cli.__name__ + ".load_images")
+    @mock.patch(cli.__name__ + ".open")
+    def test_start_with_dyno_defaults(self, _ignore_load_images, _ignore_open):
+        """
+        GIVEN a mocked CLI which does not actually load images
+        WHEN the CLI is called with the --dyno flag
+        THEN the generated Docker Compose file contains the defaults for Dyno
+        """
+        docker_compose_yml = stringIO()
+        with mock.patch.dict(LocalSetup.SUPPORTED_VERSIONS, {'master': '8.0.0'}):
+            setup = LocalSetup(argv=self.common_setup_args + ["master", "--all", "--dyno"])
+            setup.set_docker_compose_path(docker_compose_yml)
+            setup()
+        docker_compose_yml.seek(0)
+        received = yaml.safe_load(docker_compose_yml)
+        got = received['services']['dyno']
+        want = {
+                'build':
+                {
+                    'args': [],
+                    'context': 'docker/dyno',
+                    'dockerfile': 'Dockerfile'
+                },
+                'container_name': 'dyno',
+                'environment': {'TOXI_HOST': 'toxi', 'TOXI_PORT': '8474'},
+                'healthcheck': {
+                    'interval': '10s',
+                    'retries': 12,
+                    'test': [
+                        'CMD',
+                        'wget',
+                        '-T',
+                        '3',
+                        '-q',
+                        '--server-response',
+                        '-O',
+                        '/dev/null',
+                        'http://localhost:8000/'
+                        ]
+                    },
+                'ports': ['9000:8000'],
+                'volumes': ['/var/run/docker.sock:/var/run/docker.sock']}
+        self.assertDictEqual(got, want)
+
+    @mock.patch(cli.__name__ + ".load_images")
+    @mock.patch(cli.__name__ + ".open")
+    def test_start_with_toxi(self, _ignore_load_images, _ignore_open):
+        """
+        GIVEN a mocked CLI which does not actually load images
+        WHEN the CLI is called with the --dyno flag
+        THEN the generated Docker Compose file contains a configuration block for the Toxi container
+        """
+        docker_compose_yml = stringIO()
+        with mock.patch.dict(LocalSetup.SUPPORTED_VERSIONS, {'master': '8.0.0'}):
+            setup = LocalSetup(argv=self.common_setup_args + ["master", "--all", "--dyno"])
+            setup.set_docker_compose_path(docker_compose_yml)
+            setup()
+        docker_compose_yml.seek(0)
+        got = yaml.safe_load(docker_compose_yml)
+        self.assertIn('dyno', got['services'])
```

Comment:
Is this correct? I can see that this test block is exactly the same as https://github.com/elastic/apm-integration-testing/pull/1028/files#diff-99db2e84ec0f1793d9dea09793eb615b3cb0a9c8ec662380acdf530414493ae3R1190-R1203
```suggestion
        self.assertIn('toxi', got['services'])
```

Reply:
I think you're correct here. Well spotted!